### DISCUSSION TITLE
Configuring nohardlinks

### DISCUSSION THREAD

**Discussion by DMJoh** [StuffAnThings/qbit_manage#193] [Q&A] [answered]
I have radarr and sonarr set up and would like to use qbit_manage to remove the torrents which are deleted from sonarr/radarr.

I'm trying to deploy qbit_manage in a docker container using docker-compose. But I have a question about the category under `nohardlinks:`. From my limited understanding, is this the name of the qbittorrent post-download/export category?

I always get the following error, when I start qbit_manage.

```sh
Commands found in config/config.yml, ignoring env variables and using config commands instead.    
Config Error: Category *complete* is defined under nohardlinks attribute
```

I think I am doing something wrong here. Can someone assist me on this ?

My `config.yml`

```sh
commands:
  # The commands defined below will IGNORE any commands used in command line and docker env variables.
  dry_run: False
  cross_seed: False
  recheck: False
  cat_update: False
  tag_update: False
  rem_unregistered: False
  tag_tracker_error: False
  rem_orphaned: False
  tag_nohardlinks: true
  skip_cleanup: False

qbt:
  # qBittorrent parameters
  host: "192.168.0.10:8080"
  user: "admin"
  pass: "password"

directory:
  root_dir: "/media/downloads"

nohardlinks:
    completed:
      cleanup: true
```

Sorry, I am a bit confused

**Comment by DMJoh** [comment, marked as answer]
Sorry for the mess here.

I missed the part to add `cat:` and  `tracker` which was the root cause for the issue. It's my bad. Apologies.

**Comment by Shark1979xxx** [comment]
I try to do the same thing as you. Only tag noHL torrents. Could you share your config.yml file? Thanks.
